### PR TITLE
Update dep to remove vulnerabilities + add git build workflows

### PR DESCRIPTION
**Pull request by yossipapi**

### CODE REVIEW

File: .github/workflows/docker_build.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      matrix:
+        combinations:
+          - { env: "nvq2", yaml-key: "search:search.containers.search.image.tag", val-file: "regions/us-east-1/nvq2/"}
+    uses: kaltura/ovp-pipelines/.github/workflows/helm.yml@3.5.7
+    with:
+      is-local: true
+      chart-dir-location: deployment
+      applications-yaml-keys: ${{ matrix.combinations.yaml-key }}
+      values-file-location: ${{ matrix.combinations.val-file }}
+      app-repo: ovp-genie
```
Comment (ido123ziv) [inline .github/workflows/docker_build.yml:44]: please change the `app-repo`

File: .github/workflows/docker_build_manual.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      matrix:
+        combinations:
+          - { env: "nvq2", yaml-key: "search:search.containers.search.image.tag", val-file: "regions/us-east-1/nvq2/"}
+    uses: kaltura/ovp-pipelines/.github/workflows/helm.yml@3.5.7
+    with:
+      is-local: true
+      chart-dir-location: deployment
+      applications-yaml-keys: ${{ matrix.combinations.yaml-key }}
+      values-file-location: ${{ matrix.combinations.val-file }}
+      app-repo: ovp-genie
```
Comment (ido123ziv) [inline .github/workflows/docker_build_manual.yml:47]: same

**Review by ido123ziv — CHANGES_REQUESTED** [review]
please make sure the app name matches :)

File: .github/workflows/docker_build_manual.yml (hunk 2 above)
Comment (yossipapi) [reply .github/workflows/docker_build_manual.yml]: Done

File: .github/workflows/docker_build.yml (hunk 1 above)
Comment (yossipapi) [reply .github/workflows/docker_build.yml]: Done

**Review by ido123ziv — APPROVED** [review]
looks promising

**Pull request merged**